models/article.model.js

```diff
@@ -1,18 +1,39 @@
 const db = require('../db/connection');
 
 
+
 //ARTICLES ENDPOINT
-exports.selectArticles = ()=>{
-   const queryString = 
+exports.selectArticles = (topicQuery,sort_by ='created_at',order = 'desc')=>{
+    validSortByQueries = ['title','topic','author','created_at','votes','comment_count'];
+    validOrderByQueries = ['asc','desc']
+    
+    if(!validSortByQueries.includes(sort_by) || !validOrderByQueries.includes(order)){
```

Comment:
Like the way you've handled if the sortby or order isn't valid together before you get to the DB query